### PR TITLE
Fix list stretch tags

### PR DESCRIPTION
**Pull request by mraspaud**
This PR fixes the scale and offset being written as the repr of DataArrays when the stretch arguments are lists.

Also we update the link to the rainbow article.

 - [x] Tests added (for all bug fixes or enhancements)
 - [x] Tests passed (for all non-documentation changes)
 - [x] Passes ``git diff origin/master **/*py | flake8 --diff`` (remove if you did not edit any Python files)

### CODE REVIEW

File: trollimage/xrimage.py [hunk 1]
```diff
@@ -456,7 +457,7 @@ def rio_save(self, filename, fformat=None, fill_value=None,
             overviews_minsize (int): Minimum number of pixels for the smallest
                 overview size generated when `overviews` is auto-generated.
                 Defaults to 256.
-            overviews_resampling (str): Resampling method
+             (str): Resampling method
```
Comment (djhoese) [inline trollimage/xrimage.py:460]: What happened here.

File: trollimage/xrimage.py [hunk 2]
```diff
@@ -541,6 +542,10 @@ def rio_save(self, filename, fformat=None, fill_value=None,
             tags['scale'], tags['offset'] = invert_scale_offset(scale, offset)
 
         # FIXME add metadata
+        # import rioxarray
+        # import threading
+        # return data.rio.to_raster(filename, tiled=True, lock=threading.Lock())
```
Comment (djhoese) [inline trollimage/xrimage.py:547]: I know what this is, but yeah, needs to be removed.

**Review by djhoese — CHANGES_REQUESTED** [review]
Looks good except for the weird left over things and accidental removal of a docstring parameter.

File: trollimage/xrimage.py (hunk 2 above)
Comment (mraspaud) [reply trollimage/xrimage.py:547]: ah, sorry, missed that!

File: trollimage/xrimage.py (hunk 1 above)
Comment (mraspaud) [reply trollimage/xrimage.py:460]: sorry...

**Comment by mraspaud** [comment]
Sorry, that was the friday rush, trying to finish everything.

**Review by djhoese — APPROVED** [review]

**Pull request merged**